Split MQTT options and reconfigure flow (#36397)
home-assistant/home-assistant.github.io: source/_integrations/mqtt.markdown

## Configure MQTT options

To change the options, follow these steps:

1. Go to **{% my integrations title="Settings > Devices & services" %}**.
2. Select the MQTT integration.
3. Select **Configure**, then **Re-configure MQTT**.
4. To open the MQTT options page, select **Next**.

### Change MQTT discovery options

The MQTT discovery options can be changed by following these steps:

> 1. Navigate to **Settings** > **Devices & Services**

Reviewer: ```suggestion
1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
```

> 2. Find the MQTT integration and click on it

Reviewer: ```suggestion
2. Find the MQTT integration and select it.
```

> 3. Click **CONFIGURE MQTT OPTIONS** button to open the MQTT discovery options page

Reviewer: ```suggestion
3. To open the MQTT discovery options page, select the **Configure MQTT Options** button.
```

> 

Author: The MQTT integration as a dedicated page for trouble shooting and allowing to start the option flow. But the option flow now only allows to set options, and no longer allows to reconfigure the broker settings. This is now supported via the `Reconfiguration flow`.
